parquet-column/src/main/java/org/apache/parquet/column/values/bitpacking/ByteBitPackingValuesReader.java

```diff
@@ -1,14 +1,14 @@
-/* 
+/*
  * Licensed to the Apache Software Foundation (ASF) under one
  * or more contributor license agreements.  See the NOTICE file
  * distributed with this work for additional information
  * regarding copyright ownership.  The ASF licenses this file
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- * 
+ *
  *   http://www.apache.org/licenses/LICENSE-2.0
- * 
```

Comment:
Please remove all non-functional changes, like these that modify whitespace. While I like having cleaner code, we don't want to introduce unnecessary commit conflicts.

Reply:
This was an annoying side-effect of using the IDE. Is there any convenient way to change this on github, or do I have to do the whole thing all over again?

Reply:
I think I got them all.